common/src/main/kotlin/org/wycliffeassociates/otter/common/domain/narration/ChapterRepresentation.kt

```diff
@@ -0,0 +1,216 @@
+package org.wycliffeassociates.otter.common.domain.narration
+
+import com.fasterxml.jackson.core.type.TypeReference
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.reactivex.subjects.PublishSubject
+import org.slf4j.LoggerFactory
+import org.wycliffeassociates.otter.common.audio.AudioFileReader
+import org.wycliffeassociates.otter.common.data.audio.VerseMarker
+import org.wycliffeassociates.otter.common.data.workbook.Chapter
+import org.wycliffeassociates.otter.common.data.workbook.Chunk
+import org.wycliffeassociates.otter.common.data.workbook.Workbook
+import org.wycliffeassociates.otter.common.domain.audio.OratureAudioFile
+import java.io.File
+import java.io.RandomAccessFile
+import java.lang.IllegalStateException
+import kotlin.math.min
+
+private const val ACTIVE_VERSES_FILE_NAME = "active_verses.json"
+private const val CHAPTER_NARRATION_FILE_NAME = "chapter_narration.pcm"
+
+internal class ChapterRepresentation(
```

Comment:
Could we add documentation to describe this class? It'd be helpful to grab the general idea when encountering this abstract name